scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -211,6 +219,7 @@ BA_ "GenMsgCycleTime" BO_ 413 10;
 BA_ "GenMsgCycleTime" BO_ 500 100;
 BA_ "GenMsgCycleTime" BO_ 501 5000;
 BA_ "GenMsgCycleTime" BO_ 503 10;
+BA_ "GenMsgCycleTime" BO_ 505 1000;
```

Comment:
Forgot to add period to DIM_ERRORS earlier and it was causing compilation issues